<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstConfig</class>
 <widget class="QWidget" name="FirstConfig">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Première installation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Première configuration de &lt;span style=&quot; font-weight:600;&quot;&gt;EDSystem&lt;/span&gt;.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;Il est nécessaire de procéder à la configuration de la base de données afin de pouvoir utiliser le logiciel.&lt;br/&gt;Ce dernier utilise une base de donnée MySQL/MariaDB.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;Lors de cette première configuration de la BDD, le système va créer les différentes tables nécessaires au bon fonctionnement du logiciel.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;Si pour une raison ou une autre vous avez un soucis avec la BDD et qu'il faut la recréer, le menu principal vous permet de revenir à cet écran.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Attention ! &lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Cet écran supprime les tables de la base et les recrées totalement,&lt;br/&gt;les données présentes dans la base seront donc perdue.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Merci de ne pas refaire cette configuration sauf si vous devez recréer totalement la base.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Adresse hôte :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="hostEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nom d'utilisateur de la BDD :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="usernameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Mot de passe de la BDD :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Port de la BDD :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="portEdit">
           <property name="inputMask">
            <string>00000</string>
           </property>
           <property name="text">
            <string>3306</string>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Nom de la BDD :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dbnameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="testConnectionButton">
         <property name="text">
          <string>Tester</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;L'hôte est l'adresse qui permet d'accéder au serveur MySQL.&lt;br/&gt;Le nom d'utilisateur, correspond à un utilisateur du serveur MySQL.&lt;br/&gt;Le mot de passe, correspond au mot de passe de l'utilisateur MySQL.&lt;br/&gt;Le port, correspond au numéro du port utilisé pour joindre le serveur, 3306 par défaut.&lt;br/&gt;Le nom de la base, correspond au nom d'une base de donnée MySQL.&lt;/p&gt;&lt;p&gt;L'utilisateur doit disposer des droits pour créer,&lt;br/&gt;modifier et supprimer des tables SQL dans la base de donnée indiquée.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>49</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="validateButton">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
